fuel-vm/src/interpreter/blockchain/code_tests.rs

```diff
@@ -14,7 +14,7 @@ use crate::{
 use fuel_tx::Contract;
 
 #[test]
-fn test_load_contract() -> IoResult<(), Infallible> {
+fn test_load_contract_in_script() -> IoResult<(), Infallible> {
```

Comment:
Shouldn't these tests be against the `Interpreter` instead of against `LoadContractCodeCtx`?

Reply:
They should probably be against both. I still hate that `LoadContractCodeCtx` and the others like that even exist.

Reply:
Okay. I don't you to fix the architecture in this PR.